prowler/providers/aws/services/elasticache/elasticache_service.py

```diff
@@ -41,6 +41,10 @@ def _describe_cache_clusters(self, regional_client):
                             auto_minor_version_upgrade=cache_cluster.get(
                                 "AutoMinorVersionUpgrade", False
                             ),
+                            engine_version=cache_cluster.get("EngineVersion", "None"),
```

Comment:
2 things here:
1. If the default value "None" reach to the check it will be: `int("None")` which is a problem,
2. Why not store this values as a floating point instead a str?